### DISCUSSION TITLE
unable to compile rustdesk web

### DISCUSSION THREAD

**Discussion by karthikindia** [rustdesk/rustdesk#935] [Q&A]
https://github.com/rustdesk/rustdesk/tree/master/flutter/web

error screenshot - https://pix.cobrasoftwares.org/image/xpdG

**Comment by TBG-FR** [comment]
Did you manage to do it ? We should be able to build the web client from the current sources, but that requires a bit of effort... 

🔗  #2535 https://github.com/rustdesk/rustdesk/issues/4579#issuecomment-1581971865 #3742 etc.
